config/initializers/datadog.rb

```diff
@@ -43,8 +35,6 @@
   # Configuring tracing
 
   c.tracing.report_hostname = true
-  c.tracing.distributed_tracing.propagation_inject_style << 'tracecontext'
-  c.tracing.distributed_tracing.propagation_extract_style << 'tracecontext'
```

Comment:
Regarding this, the [upgrade doc](https://github.com/DataDog/dd-trace-rb/blob/release/docs/UpgradeGuide2.md#propagation-default) says:

> B3 propagation has been removed from the default propagation for distributed tracing. If you want to configure B3 propagation, see [document](GettingStarted.md#distributed-header-formats).

| 1.x                               | 2.0                    |
| --------------------------------- | ---------------------- |
| `datadog,b3multi,b3,tracecontext` | `datadog,tracecontext` |

I wasn't sure if we need b3 or b3multi, but it used to be enabled by default. It seems clear that at the very least `tracecontext` is a default so we don't need to add it.